packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -599,32 +569,32 @@ class _OutlinedButtonDefaultsM3 extends ButtonStyle {
 
   @override
   WidgetStateProperty<Size>? get maximumSize =>
-    const MaterialStatePropertyAll<Size>(Size.infinite);
+      const MaterialStatePropertyAll<Size>(Size.infinite);
 
   @override
   WidgetStateProperty<BorderSide>? get side =>
-    WidgetStateProperty.resolveWith((Set<WidgetState> states) {
-    if (states.contains(WidgetState.disabled)) {
-      return BorderSide(color: _colors.onSurface.withOpacity(0.12));
-    }
-    if (states.contains(WidgetState.focused)) {
-      return BorderSide(color: _colors.primary);
-    }
-    return BorderSide(color: _colors.outline);
-  });
+      WidgetStateProperty.resolveWith((Set<WidgetState> states) {
+        if (states.contains(WidgetState.disabled)) {
+          return BorderSide(color: _colors.onSurface.withOpacity(0.12));
+        }
+        if (states.contains(WidgetState.focused)) {
+          return BorderSide(color: _colors.primary);
+        }
+        return BorderSide(color: _colors.outline);
+      });
 
   @override
   WidgetStateProperty<OutlinedBorder>? get shape =>
-    const MaterialStatePropertyAll<OutlinedBorder>(StadiumBorder());
+      const MaterialStatePropertyAll<OutlinedBorder>(StadiumBorder());
```

Comment:
i ran dart format and now theres a bunch of these format changes 😿

Reply:
you may need to update `flutter update-packages`